variables.tf

```diff
@@ -65,6 +65,12 @@ variable "manage_aws_auth_configmap" {
   default     = true
 }
 
+variable "enable_cluster_windows_support" {
+  description = "Determines whether to create the amazon-vpc-cni configmap and windows worker roles into aws-auth."
+  type        = bool
+  default     = false
+}
+
```

Comment:
This is not necessary at the root module, right? And in any case, it's not passed down to the child module.